app/src/foss/java/com/google/android/gms/maps/MapView.java

```diff
@@ -53,8 +58,14 @@ public void getMapAsync(OnMapReadyCallback callback) {
 
   private void setDefaultConfiguration(Context context) {
     String userAgent = String.format("%s/%s", context.getString(R.string.app_name), BuildConfig.VERSION_NAME);
```

Comment:
Set the user agent to simply ```context.getPackageName()```.

This was done by the original author of the PR, but better to not leak the version and title of the app.